src/app/pages/components/tree/input.component.ts

```diff
@@ -0,0 +1,68 @@
+import { Component, OnInit } from '@angular/core';
+import { NgbActiveModal } from '@ng-bootstrap/ng-bootstrap';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+
+@Component({
+    selector: 'ngx-modal',
```

Comment:
This file should be indented with 2 spaces, not 4

Reply:
@SalahudinMalik could you check?

Reply:
@SalahudinMalik still 4 spaces